pkg/connector/mock/builder.go

```diff
@@ -44,8 +44,7 @@ func (b Builder) Init(c connector.Connector, id string, cfg connector.Config) er
 		m.EXPECT().Type().Return(connector.TypeSource).AnyTimes()
 		m.EXPECT().ID().Return(id).AnyTimes()
 		m.EXPECT().Config().Return(cfg).AnyTimes()
-		m.EXPECT().SetProvisionedBy(connector.ProvisionTypeAPI).AnyTimes()
-		m.EXPECT().ProvisionedBy().Return(connector.ProvisionTypeAPI).AnyTimes()
+		m.EXPECT().ProvisionedBy().AnyTimes()
```

Comment:
I'm wondering why this was changed? I assume calling `ProvisionedBy` would error out right now because the mock won't know what to return.
```suggestion
		m.EXPECT().ProvisionedBy().Return(connector.ProvisionTypeAPI).AnyTimes()
```